Samples/nuget.config

```diff
@@ -13,6 +13,7 @@
 
         <!-- a local directory to allow testing nupkg files without pushing to a remote feed -->
         <add key="localpackages" value="localpackages" />
+        <add key="Package source" value="https://packagesource" />
```

Comment:
This package source pointing to a non-existent domain should probably not be here. 😄